### DISCUSSION TITLE
Strawberry on mac : scanning collection is not working anymore

### DISCUSSION THREAD

**Discussion by phwuil** [strawberrymusicplayer/strawberry#1378] [General]
Hello,

Using strawberry for Mac (M2 Ultra, Mac Os Sonoma 14.3.1), I cannot scan the library (a simple unique folder with several files) :

![image](https://github.com/strawberrymusicplayer/strawberry/assets/5980636/0fb93f71-4c5d-4a91-b35f-3a59065344ae)

Blocked at 0% ...

I tried with several versions from 1.0.0 to the last (1.0.9) to HEAD... 

Do you have any clue ? 
Thanks !

**Comment by jonaski** [comment]
Usually this issue occurs when the tagreader executable is missing, or is blocked. There is an executable `strawberry.app/Contents/PlugIns/strawberry-tagreader` that does the reading of tags.
1.0.23 is the latest, not 1.0.9. Are you building yourself, or using the official macOS releases? If using the releases, only 1.0.21 and later are properly codesigned. With HEAD do you mean the latest code from master branch?
If you are building yourself, make sure you do make install, otherwise the tagreader executable is not installed.
